modules/ztp-image-based-upgrade.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept_{context}"]
+= Image-based upgrade for {sno} clusters with the {lcao}
+
+You can upgrade the platform version of the managed {sno} clusters with the {lcao}.
+By using generated a seed image, this upgrade process allows you to skip multiple minor releases and it is faster than the standard upgrade method.
+
+You generate the seed image from a cluster that is upgraded and configured with the desired {product-title} and applications by using the standard upgrade method. This is the seed cluster.
+
+[IMPORTANT]
+====
+The image-based upgrade uses custom images that are specific to the hardware platform the clusters are running on.
+Each different hardware platform requires a separate custom image.
+====
+
+With this seed image, you can upgrade the platform version on any {sno} that matches the image's cluster profile and then restore applications.
+
+//How do we specify the hardware exactly? Is it baked into the seed image based on the seed cluster we're using?
+
+The {lcao} uses two custom resources (CRs):
+
+* The `SeedGenerator` CR specifies the platform version of the seed cluster that you want to use to upgrade the target clusters.
```

Comment:
well, it specifies the name of the container image that is going to be created based on the seed cluster configuration. This container image will eventually be pushed to the registry once the image is completed.

example:

~~~yaml
apiVersion: lca.openshift.io/v1alpha1
kind: SeedGenerator
metadata:
  name: seedimage
  namespace: openshift-lifecycle-agent
spec:
  seedImage: quay.io/alosadag/ibu-seed-sno0:v4.14.1
~~~


It will be pushed to my quay.io account. However, the naming depends on the user. I recommend <seed-cluster-name>:<ocp_version>.